### PR TITLE
27 component link

### PR DESCRIPTION
**Pull request by lulek1410**

### CODE REVIEW

**Comment by lulek1410** [comment]
it seem to be failing on ci. I dont really know what is the problem here. Could you plese look at this @daniknewgarden?

File: src/lib/components/Link/Link.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const StyledLink = styled.a<LinkProps>`
+	font-size: 14px;
+	font-weight: 700;
+	color: ${({ theme }) => theme.colors.primary.main};
+
+	cursor: default;
+	border-radius: 4px;
+	padding: 4px 4px;
+
+	:is(&:hover, &:focus, &:active) {
```
Comment (daniknewgarden) [inline src/lib/components/Link/Link.tsx:15]: I think it's better to have the following behavior:
1. On hover change only color. 
2. On active, focus change background, don't change the color.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: Testing
-on: push
```
Comment (daniknewgarden) [inline .github/workflows/tests.yml:2]: I've already made this change in the main branch. Could you please rebase against it?

File: src/lib/components/Link/Link.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Link } from "./Link";
+
+const meta = {
+	title: "Link",
```
Comment (daniknewgarden) [inline src/lib/components/Link/Link.stories.tsx:5]: Please replace with Base/Link

**Review by daniknewgarden — CHANGES_REQUESTED** [review]
Please fix the comments, and maybe we can merge it. I'll fix the pipeline. 
Thank you 🥺

File: src/lib/components/Link/Link.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	border-radius: 4px;
+	padding: 4px 4px;
+
+	&:hover {
+		color: ${({ theme }) => theme.colors.primary.lighter};
+	}
+
+	&:focus,
+	&:active {
+		background-color: ${({ theme }) => theme.colors.background.light[500]};
```
Comment (daniknewgarden) [inline src/lib/components/Link/Link.tsx:21]: You can put outline rules to this place.

**Review by daniknewgarden — APPROVED** [review]
Approved with one small comment.

File: src/lib/components/Link/Link.tsx (hunk 4 above)
Comment (lulek1410) [reply src/lib/components/Link/Link.tsx:21]: will do in sort of post mege commit since this one is approved

**Pull request merged**